raiden_contracts/tests/test_service_registry.py

```diff
@@ -2,49 +2,80 @@
 
 import pytest
 from eth_tester.exceptions import TransactionFailed
-from web3.contract import Contract
+from web3 import Web3
+from web3.contract import Contract, get_event_data
 
-from raiden_contracts.tests.utils.constants import CONTRACT_DEPLOYER_ADDRESS
+from raiden_contracts.constants import CONTRACT_SERVICE_REGISTRY, EVENT_REGISTERED_SERVICE
+from raiden_contracts.contract_manager import ContractManager, contracts_precompiled_path
+from raiden_contracts.tests.utils.constants import EMPTY_ADDRESS, SERVICE_DEPOSIT
 
 
-def test_owner_of_service_registry(service_registry: Contract) -> None:
-    """ The owner of ServiceRegistry should be the deployer """
-    assert service_registry.functions.owner().call() == CONTRACT_DEPLOYER_ADDRESS
+def test_deposit_contract(
+    get_deposit_contract: Callable, custom_token: Contract, get_accounts: Callable
+) -> None:
+    (A,) = get_accounts(1)
```

Comment:
nit: You don't need braces here

Reply:
I like it without the braces, but it's with the braces everywhere in the codebase.  For now I just follow the local convention.